corehq/apps/users/views/utils.py

```diff
@@ -206,3 +206,14 @@ def get_user_location_info(domain, user_location_ids, user_id):
         'orphaned_case_count_per_location': orphaned_case_count_per_location,
         'shared_locations': shared_locations,
     }
+
+
+def filter_user_query_by_locations_accessible_to_user(user_es, domain, couch_user):
+    if not couch_user.has_permission(domain, 'access_all_locations'):
+        loc_ids = (SQLLocation.objects.accessible_to_user(domain, couch_user).location_ids())
+        user_es = user_es.location(list(loc_ids))
+    return user_es
```

Comment:
I think this is missing the `mobile_users()` filter that was here before, so now web users are showing up in the mobile workers list on staging.